Python/codegen.c

```diff
@@ -1241,10 +1239,12 @@ codegen_function_body(compiler *c, stmt_ty s, int is_async, Py_ssize_t funcflags
     RETURN_IF_ERROR(
         codegen_enter_scope(c, name, scope_type, (void *)s, firstlineno, NULL, &umd));
 
+    PySTEntryObject *ste = SYMTABLE_ENTRY(c);
     Py_ssize_t first_instr = 0;
     PyObject *docstring = _PyAST_GetDocString(body);
```

Comment:
Can `*docstring = _PyAST_GetDocString(body);` move to inside the `if (ste->ste_has_docstring) {` block?